Gemfile

```diff
@@ -3,3 +3,8 @@ source "http://rubygems.org"
 
 # Specify your gem's dependencies in rate-limiting.gemspec
 gemspec
+
+gem "pry"
```

Comment:
Can we denote these as development dependencies by putting them in a `:development` group?